appveyor.yml

```diff
@@ -1,9 +1,5 @@
 environment:
   matrix:
-    - PYTHON: "C:\\Python37"
-      PYTHON_VERSION: 3.7
-      PYTHON_ARCH: 32
-
```

Comment:
Why did this happen?

Reply:
@SantiagoTorres I had committed for another unrelated documentation issue but neglected to change branches before I did. I also wanted to sync my forked repo so as to keep it up-to-date with the original repo, hence the failed merge.